po/pt-BR.po

```diff
@@ -7013,11 +7074,11 @@ msgstr ""
 "     }\n"
 "     fn create_default() {\n"
 "         let tmp = Pessoa {\n"
-"             ..Default::default()\n"
+"             ..Pessoa::default()\n"
```

Comment:
This should match the struct definition above:

```suggestion
"             ..Person::default()\n"
```

Reply:
Actually I had translated the struct name in a previous PR (Person -> Pessoa)